events/event_nettrace.go

```diff
@@ -15,6 +19,56 @@ var NetTraceIntegrator = func() func(basictracer.SpanEvent) {
 			tr = trace.New("tracing", t.OperationName)
 		case basictracer.EventFinish:
 			tr.Finish()
+		case basictracer.EventTag:
+			tr.LazyPrintf("%s:%v", t.Key, t.Value)
+		case basictracer.EventLogFields:
+			var buf bytes.Buffer
+
+			// Search for an "event" or "error" field for the "main" message.
```

Comment:
my two cents: this change arrived at an awkward time since the KV logging stuff is (too) fresh and we haven't standardized on idioms for most of these keys... I would probably suggest simply recording all of the key:value pairs blindly and indiscriminately for now, adding the sort of logic you have here once we actually have declared constants (a la the `ext` package) for soon-to-be standard log keys.